### PR TITLE
Added linear search implementation in Python

### PR DESCRIPTION
**Pull request by singhpratyush**

### CODE REVIEW

File: linear_search/linear_search.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    return None
+
+
+def main():
+    arr = range(100)
+    index = linear_search(arr, 55)
+    if index is None:
+        print('Element not found in array')
+    else:
+        print('Element found at index %s' %index)
```
Comment (aviaryan) [inline linear_search/linear_search.py]: space after % needed.

**Comment by aviaryan** [comment]
Hey @singhpratyush , it would be great if you can add an entry for linear search in the README, in the "Implemented Algorithms" section.

Eg >

* [Linear Search](https://en.wikipedia.org/wiki/Linear_search) ([Python](http://link.to/code/python))

**Comment by singhpratyush** [comment]
Yes, would be better that way.

**Comment by singhpratyush** [comment]
I have added it to `README` and synced with main branch.

**Pull request merged**